lib/steep/drivers/watch.rb

```diff
@@ -129,7 +129,11 @@ def run()
         end
 
         listener.stop
-        main_thread.join
+        begin
+          main_thread.join
+        rescue Interrupt
+          master.interrupt unless main_thread.join(5)
```

Comment:
After `Interrupt`, wait for `main_thread` again with timeout value.
If the timeout was occured, `master.intterupt` was called because  `Thread#join` returns `nil`.